Makefile

```diff
@@ -30,6 +33,10 @@ help:
 addsite:
 	@docker-compose exec zotonic bin/zotonic addsite -s ginger -H $(name).docker.test $(name)
 
+api-docs:
+	@yaml2json docs/rest-api.yaml > /tmp/rest-api.json
+	@spectacle -1 -t /tmp -f ginger-rest-api.html /tmp/rest-api.json
```

Comment:
Should we add a `deps` step to Ginger’s Makefile, too? Otherwise `make api-docs` doesn’t work out of the box, as it needs to be preceded by an `npm install`.